src/php/ext/grpc/php_grpc.c

```diff
@@ -222,6 +232,55 @@ void apply_ini_settings(TSRMLS_D) {
     strcat(poll_str, GRPC_G(poll_strategy));
     putenv(poll_str);
   }
+
+  if (GRPC_G(grpc_verbosity)) {
+    char *verbosity_str = malloc(sizeof("GRPC_VERBOSITY=") +
+                                 strlen(GRPC_G(grpc_verbosity)));
+    strcpy(verbosity_str, "GRPC_VERBOSITY=");
```

Comment:
How about using gpr_asprintf instead of string concatenation?

Reply:
Also noted. I can address them separately on all the other env vars.